mobilecoind/strategies/drain-accounts.py

```diff
@@ -117,13 +117,13 @@ def filename_key(filename):
 
     stub = connect(args.mobilecoind_host, args.mobilecoind_port)
     source_accounts = [
-        load_key_and_register(os.path.join(args.key_dir, k), stub)
-        for k in sorted(glob.glob(os.path.join(args.key_dir, '*.json')), key=filename_key)
+        load_key_and_register(account_key, stub)
+        for account_key in sorted(glob.glob(os.path.join(args.key_dir, '*.json')), key=filename_key)
```

Comment:
Potentially add checking for file existence
```python
        load_key_and_register(account_key, stub)
        for account_key in sorted(glob.glob(os.path.join(args.key_dir, '*.json')), key=filename_key) if os.path.isfile(account_key)
```

Reply:
i'm worried this will cover up a problem rather than making it loud, we want it to fail if the files dont exist